src/backend/expungeservice/models/charge_types/schedule_1_p_c_s.py

```diff
@@ -8,8 +8,9 @@
 class Schedule1PCS(Charge):
     type_name: str = "Schedule 1 PCS"
 
-    def _default_type_eligibility(self):
+    def _type_eligibility(self):
         if self.dismissed():
-            return TypeEligibility(EligibilityStatus.ELIGIBLE, reason="Eligible under 137.225(1)(b)")
-        else:
-            return TypeEligibility(EligibilityStatus.ELIGIBLE, reason="Eligible under 137.225(5)(C)")
+            # TODO: Are violation dismissals for a PCS actually eligible? This does not appear to be the case.
```

Comment:
Just checking, but I assume that this is tracked somewhere?

Reply:
Added a new issue, thanks